apps/meteor/tests/e2e/page-objects/fragments/home-sidenav.ts

```diff
@@ -68,6 +68,14 @@ export class HomeSidenav {
 		return this.page.locator(`[data-qa="sidebar-item"][aria-label="${name}"]`);
 	}
 
+	async selectMarkAsUnread(name: string) {
+		const sidebarItem = this.getSidebarItemByName(name);
+		await sidebarItem.focus();
+		await sidebarItem.locator('.rcx-sidebar-item__menu').click();
+		await this.page.getByRole('option', { name: 'Mark Unread' }).click();
+		return sidebarItem;
```

Comment:
I would suggest avoiding returning the sidebarItem here, and in order to get it use the `getSidebarItemByName` where you need